src/shims-tsx.d.ts

```diff
@@ -3,9 +3,9 @@ import Vue, { VNode } from 'vue'
 declare global {
```

Comment:
This "external module" is a TS-specific module system used before ES-module. It was needed in case we use .tsx files or need jsx syntax support.

ESLint had now informed me that the elements below are a duplicate declaration. The other declaration is in node_modules/vue/types/jsd.d.ts.

Therefore, I believe this file is obsolete. (I don't see anything different in VS Code.)